Makefile.build

```diff
@@ -369,7 +369,7 @@ grammar/grammar.cma : $(GRAMCMO)
 	$(SHOW)'OCAMLC -a $@'
 	$(HIDE)$(GRAMC) $^ -linkall -a -o $@
 
-grammar/coqpp: $(COQPPCMO) grammar/coqpp_main.ml
+$(COQPP): $(COQPPCMO) grammar/coqpp_main.ml
 	$(SHOW)'OCAMLC -a $@'
 	$(HIDE)$(GRAMC) $^ -linkall -o $@
```

Comment:
>@Zimmi48 it looks like it's performing a nonsense action, namely OCAMLC -a bin/coqpp

It's from here. The SHOW is lying about what is done.

Reply:
It doesn't explain why it fails though. The compilation command looks legit and it does work with other CI targets.